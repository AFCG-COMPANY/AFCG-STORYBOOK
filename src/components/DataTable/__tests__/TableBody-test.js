import React from 'react';
import { mount } from 'enzyme';
import { Table, TableBody } from '../';

describe('DataTable.TableBody', () => {
  it('should render', () => {
    const wrapper = mount(
      <Table>
        <TableBody className="custom-class" />
      </Table>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
