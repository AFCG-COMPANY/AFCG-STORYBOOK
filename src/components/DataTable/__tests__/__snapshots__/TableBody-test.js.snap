// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataTable.TableBody should render 1`] = `
<Table>
  <table
    className="bx--data-table-v2 bx--data-table-v2--zebra"
  >
    <TableBody
      className="custom-class"
    >
      <tbody
        className="custom-class"
      />
    </TableBody>
  </table>
</Table>
`;
